tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true
   },
-  "exclude": ["node_modules", "assets", "integration-tests", "build", "dist", "static", "config"],
+  "exclude": ["node_modules", "assets", "integration-tests", "build", "dist", "static", "config", "manage-accounts-specs"],
```

Comment:
is this necessary?

Reply:
yes, because the groovy tests contain a `build/reports` directory that contains javascript that was being validated